setup/www/resources/config/nodejs.org

```diff
@@ -40,7 +40,11 @@ server {
     default_type text/plain;
     index index.html;
 
+    error_page 404 @localized_404;
+
     location / {
+        try_files $uri $uri/ @english_fallback;
```

Comment:
Hmmm, might have been from my investigations. Can't recall why/if it would be necessary.